contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/video-codec/jobs.pxu

```diff
@@ -75,3 +75,23 @@ flags: also-after-suspend
 environ: GST_LAUNCH_BIN, PERFORMANCE_PYTHON_MODULE_PATH, PERFORMANCE_FUNCTION_NAME
 command:
     gst_v4l2_video_decoder_performance.py -gp {{golden_sample_file}} -dp {{decoder_plugin}} -mf {{minimum_fps}} -pmt "{{performance_target}}"
+
+unit: template
+template-engine: jinja2
+template-resource: video_codec_resource
+template-filter:
+    video_codec_resource.scenario == "gst_encoder_psnr"
+    ( video_codec_resource.platform == "genio-350" or video_codec_resource.platform == "genio-510" or video_codec_resource.platform == "genio-700" or video_codec_resource.platform == "genio-1200" )
+template-unit: job
+template-id: ce-oem-video-codec/gst_encoder_psnr
+_template-summary: To check if the PSNR
+id: ce-oem-video-codec/{{ scenario }}-{{ encoder_plugin }}-{{ width }}x{{ height }}@{{ framerate }}fps-{{ color_space }}-{{ mux }}
+_summary: Encoder PSNR - {{ encoder_plugin }}-{{ width }}x{{ height }}@{{ framerate }}fps-{{ color_space }}-{{ mux }}
+_description: Test if the PSNR value of an artifact which is generated from {{ encoder_plugin }} encoder reaches the acceptable threshold
```

Comment:
```suggestion
template-id: ce-oem-video-codec/geino-gst_encoder_psnr
_template-summary: To verify the PSNR ratio meets requirement on Geino platform
id: ce-oem-video-codec/{{ scenario }}-{{ encoder_plugin }}-{{ width }}x{{ height }}@{{ framerate }}fps-{{ color_space }}-{{ mux }}
_summary: Encoder PSNR - Geino - {{ encoder_plugin }}-{{ width }}x{{ height }}@{{ framerate }}fps-{{ color_space }}-{{ mux }}
_description: Test if the PSNR value of an artifact which is generated from {{ encoder_plugin }} encoder reaches the acceptable threshold on Geino platform
```

Reply:
These test cases are designed for the Geino platform, so it would be better to have an identical name.

Reply:
See https://github.com/canonical/checkbox/pull/1491/commits/b180019c13e9f9cb511d5f19f6338f9a2ab4c720